### PR TITLE
Add destination ip and port to report data

### PR DESCRIPTION
**Pull request by bianpengyuan**
**What this PR does / why we need it**:
Allow mixer to extract container name for metrics.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: include/istio/utils/attributes_builder.h [hunk 1]
```diff
@@ -104,6 +104,11 @@ class AttributesBuilder {
     }
   }
 
+  bool HasAttribute(const std::string& key) {
```
Comment (qiwzhang) [inline include/istio/utils/attributes_builder.h]: add function const.

**Comment by bianpengyuan** [comment]
@qiwzhang Done.

File: include/istio/utils/attributes_builder.h (hunk 1 above)
Comment (bianpengyuan) [reply include/istio/utils/attributes_builder.h]: Done.

**Pull request merged**